ember-flatpickr/unpublished-development-types/index.d.ts

```diff
@@ -1,4 +1,11 @@
 // Add any types here that you need for local development only.
 // These will *not* be published as part of your addon, so be careful that your published code does not rely on them!
-import "ember-source/types/stable";
-import "ember-source/types/preview";
+import 'ember-source/types/stable';
+import 'ember-source/types/preview';
+import type flatpickr from 'flatpickr';
+
+declare global {
```

Comment:
Do we need these global types anymore? Since we are not manually putting flatpickr on the window?